lib/modules/platform/gitlab/index.ts

```diff
@@ -1090,8 +1128,11 @@ export async function ensureComment({
         .replace(regEx(/PR/g), 'MR')
     : topic;
   const comments = await getComments(number);
+  const discussions = await getDiscussions(number);
```

Comment:
This seems inefficient because won't it fetch discussions unnecessarily even when blocksMerge=false?